Cache.podspec

```diff
@@ -18,4 +18,5 @@ Pod::Spec.new do |s|
   s.tvos.source_files = 'Source/{iOS,Shared}/**/*'
 
   s.frameworks = 'Foundation'
+  s.dependency 'MD5'
```

Comment:
We could reconsider the source of MD5 generation later and go wth this library to fix the current bug. What do you think @zenangst ?